Create ui-c8y-1021-0-0-edit-remove-widget-dropdown-doesnt-hide-on-saving-dashboard-and-allows.md (#2590)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-0-0-edit-remove-widget-dropdown-doesnt-hide-on-saving-dashboard-and-allows.md

---
date: ""

> title: Widget edit/remove dropdown automatically closes after saving a dashboard

Reviewer: ```suggestion
title: Widget action dropdown closes properly after saving a dashboard
```

> title: Widget edit/remove dropdown no longer hides automatically after saving dashboard

Reviewer: ```suggestion
title: Widget edit/remove dropdown no longer hides automatically after saving a dashboard
```

> title: Widget edit/remove dropdown no longer hides automatically after saving a dashboard

Reviewer: ```suggestion
title: Widget edit/remove dropdown automatically closes after saving a dashboard
```

> title: Edit/remove widget dropdown doesnt hide on saving dashboard and allows to modify it after (#7131)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Widget edit/remove dropdown no longer hides automatically after saving dashboard
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59669
version: 1021.0.0
---

> Edit/remove widget dropdown doesnt hide on saving dashboard and allows to modify it after (#7131)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, after making changes to a widget on a dashboard and saving the dashboard, the widget's edit/remove dropdown menu would remain open. This could lead to unintended modifications if the user accidentally interacted with the open dropdown. With this change, the widget's edit/remove dropdown menu now automatically closes after saving the dashboard. This prevents the dropdown from staying open and reduces the risk of inadvertent changes to the widget after the dashboard has been saved.
```

> Previously, after making changes to a widget in a dashboard and saving the dashboard, the widget's edit/remove dropdown menu remained open. This could lead to unintended modifications if the user accidentally interacted with the open dropdown. With this change, the widget's edit/remove dropdown menu now automatically closes after saving the dashboard. 

Reviewer 2: ```suggestion
Previously, opening the edit/remove dropdown and saving the dashboard, the widget's dropdown menu remained open. This could lead to unintended modifications if the user accidentally interacted with the open dropdown. With this change, the widget's edit/remove dropdown menu now automatically closes after saving the dashboard. 
```

Reviewer 2: I would say this is more clear

Reviewer: Not for me :-). What do you mean with "settings" dropdown? Is it the same as the edit/remove dropdown mentioned in the next sentence? Please be more precise.

Reviewer 2: yep, this is the same as edit/remove dropdown from the next sentence

Reviewer 2: I updated it to edit/remove dropdown, could you have a look ?

Reviewer: @igre-c8y I still think that the information "after making changes to a widget in a dashboard" is relevant. How about the following suggestion?
```suggestion
Previously, when opening the edit/remove dropdown to make changes to a widget in a dashboard and then saving the dashboard, the widget's edit/remove dropdown remained open. This could lead to unintended modifications if the user accidentally interacted with the open dropdown. With this change, the widget's action dropdown now automatically closes after saving the dashboard. 
```

Reviewer 2: true, looks good to me

> Previously, after making changes to a widget on a dashboard and saving the dashboard, the widget's edit/remove dropdown menu would remain open. This could lead to unintended modifications if the user accidentally interacted with the open dropdown. With this change, the widget's edit/remove dropdown menu now automatically closes after saving the dashboard. This prevents the dropdown from staying open and reduces the risk of inadvertent changes to the widget after the dashboard has been saved.

Reviewer: ```suggestion
Previously, after making changes to a widget in a dashboard and saving the dashboard, the widget's edit/remove dropdown menu remained open. This could lead to unintended modifications if the user accidentally interacted with the open dropdown. With this change, the widget's edit/remove dropdown menu now automatically closes after saving the dashboard. 
```
